src/resolve/font-size.ts

```diff
@@ -28,7 +28,15 @@ export default function fontSize(
 
   const { lineHeight, letterSpacing } = rest;
   if (lineHeight) {
-    mergeStyle(`lineHeight`, lineHeight, style);
+    const parsed = parseNumericValue(lineHeight);
```

Comment:
This approach is overall good, but it should be extracted out to a helper function so it can be applied to both configuration styles (see line 26) above, and the comment I left showing how to create a failing test to guide you.